tutoraspects/templates/aspects/jobs/init/clickhouse/init-clickhouse.sh

```diff
@@ -61,6 +61,30 @@ GRANT {{ ON_CLUSTER }} SELECT ON system.events TO '{{ ASPECTS_CLICKHOUSE_REPORT_
 GRANT {{ ON_CLUSTER }} SELECT ON system.metrics TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
 GRANT {{ ON_CLUSTER }} SELECT ON system.replication_queue TO '{{ ASPECTS_CLICKHOUSE_REPORT_USER }}';
 
+-- Create our named collections, these are used to back dictionaries instead of
+-- putting credentials directly into dict definitions, which allows us to change
+-- passwords without having to completely re-run Alembic migrations.
+DROP NAMED COLLECTION IF EXISTS local_ch_xapi;
+
+-- These are strictly for referencing local data, so localhost is appropriate
+CREATE NAMED COLLECTION local_ch_xapi AS
+host = 'localhost',
+port = '{{ CLICKHOUSE_INTERNAL_NATIVE_PORT }}',
+database = '{{ ASPECTS_XAPI_DATABASE }}',
+user = '{{ CLICKHOUSE_ADMIN_USER }}',
+password = '{{ CLICKHOUSE_ADMIN_PASSWORD }}',
+secure = {% if CLICKHOUSE_SECURE_CONNECTION %}1{% else %}0{% endif %};
```

Comment:
does the secure flag makes sense in a local ch?

Reply:
I'm not sure, that's what I'm going to be testing on the cloud providers.